### DISCUSSION TITLE
Control position and size of MudExDialog

### DISCUSSION THREAD

**Discussion by RDUser-dev** [fgilde/MudBlazor.Extensions#80] [Q&A] [answered]
Hi, I have an application with several inline MudExDialogs.
I need to make the position and size persistent so that they are retained when the page is refreshed.

I was thinking of using localstorage and binding it to the size and position properties of the MudExDialog, but unfortunately I don't think these properties exist.

Is there another way to control the position and size of the MudExDialog? Or, is there a better approach?
Thank you all.

**Comment by fgilde** [comment]
No currently not. In JS i have a method to set position what Iam using if you enable ShowAtCursor. However if you want to do something like this you can currently do it in JavaScript on your own, but I think I can implement a size and postion property on dialogoptionsex to handle that if you want. But then please create a feature request issue

**Reply by RDUser-dev** [reply]
Thanks for the response
Issue created  #81

**Reply by fgilde** [reply, marked as answer]
Issue implemented
